File: answers/VrishtiSharma/Conditonals/Q4.py
x,y = map(int, input("Enter two integers : ").split())
if x>0 and y>0:
    print("Quadrant 1")
elif x<0 and y>0:
    print("Quadrant 2")
elif x<0 and y<0:
    print("Quadrant 3")
elif x>0 and y<0:
    print("Quadrant 4")
elif y == 0:
    print("X axis")
elif x == 0:
    print("Y axis")
elif x == 0 and y == 0:
    print("Origin")

File: answers/Animeshbarve04/Conditions/Q4.c
#include <stdio.h>

int main()
{
    int x, y;
    //printf("x and y \n");
    scanf("%d %d", &x, &y);
    if (x == 0)
    {
        y == 0 ? printf("Origin") : printf("Y axis");
    }
    if (y == 0)
    {
        x == 0 ? printf("Origin") : printf("X axis");
    }
    if (x > 0)
    {
        y > 0 ? printf("Quadrant 1") : printf("Quadrant 4");
    }
    if (x < 0)
    {
        y > 0 ? printf("Quadrant 2") : printf("Quadrant 3");
    }

    return 0;
}